### PR TITLE
 minor changes to support testing on windows

### PR DESCRIPTION
**Pull request by poliset1**
The following changes are made to support testing on windows. 
1.  Changing single quote to double quite for system call 
2.  separating setting environment variables in windows and linux

### CODE REVIEW

File: test/xpu/extended/run_test_with_skip.py [hunk 1]
```diff
@@ -183,15 +183,22 @@
...
-test_command = "PYTORCH_TEST_WITH_SLOW=1 pytest -v test_ops_xpu.py"
-test_command += skip_options
-
-res = os.system(test_command)
-exit_code = os.WEXITSTATUS(res)
-sys.exit(exit_code)
+if os.name == "nt":
+    test_command = "pytest -v test_ops_xpu.py"
+    test_command += skip_options
+    print("running", test_command)
```
Comment (min-jean-cho) [inline test/xpu/extended/run_test_with_skip.py:195]: Please remove debug prints.

File: test/xpu/run_test_with_only.py [hunk 2]
```diff
@@ -7,35 +7,53 @@
...
             skip_options += skip_option
-        skip_options += "'"
-        test_command = (
-            "PYTORCH_ENABLE_XPU_FALLBACK=1 PYTORCH_TEST_WITH_SLOW=1 pytest -v "
-            + test_case
-        )
-        test_command += skip_options
-        return os.system(test_command)
+        skip_options += "\""
+        if os.name == "nt":
```
Comment (min-jean-cho) [inline test/xpu/run_test_with_only.py:15]: Is it better to set `os.environ["PYTORCH_ENABLE_XPU_FALLBACK"] = 1` for both Windows and Linux?

**Comment by min-jean-cho** [comment]
Please apply the same changes in `test/xpu/xpu_test_utils.py`

File: test/xpu/run_test_with_only.py [hunk 3]
```diff
@@ -6,37 +6,28 @@
...
-        test_command += exe_options
+        exe_options += "\""
+        test_command   = "pytest -v "  + test_case  + exe_options
         return os.system(test_command)
     else:
-        test_command = (
-            "PYTORCH_ENABLE_XPU_FALLBACK=1 PYTORCH_TEST_WITH_SLOW=1 pytest -v "
-            + test_case
-        )
+        test_command   = "pytest -v "
```
Comment (min-jean-cho) [inline test/xpu/run_test_with_only.py:28]: Please add `test_case`

File: test/xpu/run_test_with_only.py [hunk 4]
```diff
@@ -6,37 +6,28 @@
...
         return os.system(test_command)
     else:
-        test_command = (
-            "PYTORCH_ENABLE_XPU_FALLBACK=1 PYTORCH_TEST_WITH_SLOW=1 pytest -v "
-            + test_case
-        )
+        test_command   = "pytest -v "  
         return os.system(test_command)
-
+
```
Comment (min-jean-cho) [inline test/xpu/run_test_with_only.py:30]: Please remove.

**Review by min-jean-cho — APPROVED** [review]

**Comment by min-jean-cho** [comment]
@fengyuan14, please take a look, thanks.

File: test/xpu/extended/run_test_with_skip.py [hunk 5]
```diff
@@ -151,15 +151,16 @@
...
+skip_options = " -k \"not " + skip_list[0]
 for skip_case in skip_list[1:]:
     skip_option = " and not " + skip_case
     skip_options += skip_option
-skip_options += "'"
+skip_options += "\""
 
-test_command = "PYTORCH_TEST_WITH_SLOW=1 pytest -v test_ops_xpu.py"
-test_command += skip_options
+os.environ["PYTORCH_ENABLE_XPU_FALLBACK"]="1"
```
Comment (daisyden) [inline test/xpu/extended/run_test_with_skip.py:160]: The extended test does not expect a fallback to cpu, it is to cover all the dtypes of the op we supported. Please remove this line.

Comment (min-jean-cho) [reply test/xpu/extended/run_test_with_skip.py]: Thanks for the catch, done.

**Comment by daisyden** [comment]
Please help to ensure the Linux test result with the same number of PASSED, SKIPPED and XFAIL as before.

**Pull request merged**